README.md

```diff
@@ -123,12 +123,13 @@ This will show up on the pypi project page
 
 ### set `python_requires`
 
-A few sources are searched for guessing `python_requires`:
+If `--min-py-version` is passed, it will be used to set `python_requires`.
+
+Otherwise, the following sources are searched:
 
 - the existing `python_requires` setting itself
 - `envlist` in `tox.ini` if present
 - python version `classifiers` that are already set
-- the `--min-py3-version` argument (currently defaulting to `3.7`)
 
 If the minimum version is detected as python2, the `--min-py3-version`
 argument will be used to exclude python3.x versions (see below).
```

Comment:
this part is gone now right?

Reply:
Agreed this doc is wrong, but the behavior is not totally gone. See `test_guess_python_requires_from_classifiers` - when py2 is inferred from `python_requires`, then `--min-py-version`'s default is used to set the excluded versions